layouts/partials/footer.html

```diff
@@ -1,15 +1,15 @@
 <footer>
     <div class="subfooter newsletter">
         <div id="mc_embed_signup" class="subfooter-content">
-            <span>{{ .Params.newsletter }}</span>
+            <span>{{ i18n "newsletter" }}</span>
             <form action="{{ .Params.mailchimp }}" method="post" id="mc-embedded-subscribe-form"
```

Comment:
You sure this is working? You moved the url in the lang file, but you don't seem to use it. I might have missed something, but I thought I'd see something like that instead 
```
{{ i18n "mailchimp" }}
```